Radzen.Blazor/Rendering/Popup.razor

```diff
@@ -30,12 +28,14 @@
     [Parameter]
     public EventCallback Close { get; set; }
 
+    public bool IsOpen { get; private set; }
```

Comment:
Not sure why this change is needed. Usually properties should be parameters - it will be better to leave it as field.

Reply:
It was used by the `InlineOrPopup` component, and also by the TimeSpanPicker itself in the past. But now this change is no longer needed so I reverted it.